types.d.ts

```diff
@@ -14,3 +14,28 @@ type TenantConfig = {
         scope: string | null,
     }
 }
+
+type UploadError = {
+    status: number
+    error: string
+}
```

Comment:
`UploadError` is basically equal to `AuthError`. Why don't we just mantain one type for any kind of error ?

Reply:
used a type named ServerError